vcluster/deploy/topologies/isolated-workloads.mdx

```diff
@@ -28,48 +46,76 @@ policies:
     enabled: true
 ```
 
-For all options and more information take a look at the [policies](/docs/vcluster/configure/vcluster-yaml/policies/) documentation.
+For all options and more information take a look at the
+[policies](/docs/vcluster/configure/vcluster-yaml/policies/) documentation.
+
+:::info
+When enabling resource quotas locally, make sure to add a `--expose-local=false`
+flag to your `vcluster create [...]` command, as by default the vCluster CLI
+will try to automatically expose the vCluster using NodePorts, when interacting
+with a local Kubernetes cluster.
+:::
 
 <Policies/>
 
 ## Network Isolation
 
-Workloads created by vCluster will be able to communicate with other workloads in the host cluster through their cluster ips. This can be sometimes beneficial if you want to purposely access a host cluster service, which is a good method to share services between vClusters. However, you often want to isolate namespaces and do not want the pods running inside vCluster to have access to other workloads in the host cluster.
-This requirement can be accomplished by using [Network Policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/) for the namespace where vCluster is installed in.
+Workloads created by vCluster will be able to communicate with other workloads in
+the host cluster through their cluster ips. This can be sometimes beneficial if
+you want to purposely access a host cluster service, which is a good method to
+share services between vClusters. However, you often want to isolate namespaces
```

Comment:
update from `vClusters` to `virtual clusters`. We should never pluralize vCluster.